test_files/COS20007-Enrol-Students.csv

```diff
@@ -1 +1,13 @@
-subject_code,username,first_name,last_name,email,tutorial,,,
COS20007,student_0,Donnie,Powlowski,student_0@doubtfire.com,LA1-01,,,
COS20007,student_1,Vallie,Lehner,student_1@doubtfire.com,LA1-01
COS20007,student_2,Emmanuel,Ondricka,student_2@doubtfire.com,LA1-01
COS20007,student_3,Hettie,Hermiston,student_3@doubtfire.com,LA1-03
COS20007,student_4,Dolly,Leuschke,student_4@doubtfire.com,LA1-03
COS20007,student_5,Dedric,Brekke,student_5@doubtfire.com,LA1-03
COS20007,cliff,Cliff,Warren,cliff@doubtfire.com,LA1-03
COS20007,student_99,New Student,Name,student_99@swin.edu.au,LA1-01
COS20007,student_100,Student 100,S100 Name,student_100@swin.edu.au,LA1-99
COS20007,student_101,Student 101,S101 Name,student_101@swin.edu.au,
COS20007,student_102,Email,Wrong,student_99@swin.edu.au,LA1-01
COS20007,cliff,Cliff,Warren,cliff@doubtfire.com,LA1-03,
,,,,,
,,,,,
\ No newline at end of file
+unit_code,username,first_name,last_name,email,tutorial
+COS20007,student_0,Donnie,Powlowski,student_0@doubtfire.com,LA1-01
+COS20007,student_1,Vallie,Lehner,student_1@doubtfire.com,LA1-01
+COS20007,student_2,Emmanuel,Ondricka,student_2@doubtfire.com,LA1-01
+COS20007,student_3,Hettie,Hermiston,student_3@doubtfire.com,LA1-03
+COS20007,student_4,Dolly,Leuschke,student_4@doubtfire.com,LA1-03
+COS20007,student_5,Dedric,Brekke,student_5@doubtfire.com,LA1-03
+COS20007,cliff,Cliff,Warren,cliff@doubtfire.com,LA1-03
+COS20007,student_99,New Student,Name,student_99@swin.edu.au,LA1-01
+COS20007,student_100,Student 100,S100 Name,student_100@swin.edu.au,LA1-99
+COS20007,student_101,Student 101,S101 Name,student_101@swin.edu.au,
+COS20007,student_102,Email,Wrong,student_99@swin.edu.au,LA1-01
+COS20007,cliff,Cliff,Warren,cliff@doubtfire.com,LA1-03
```

Comment:
Keep the empty rows in here to test how that works when empty rows are imported.